cloud-controller-manager/do/loadbalancers.go

```diff
@@ -1194,6 +1220,16 @@ func getPorts(service *v1.Service, anno string) ([]int, error) {
 	return portsInt, nil
 }
 
+// getStrings returns the strings for the given service and annotation.
+func getStrings(service *v1.Service, anno string) []string {
+	vals, ok := service.Annotations[anno]
+	if !ok || len(vals) == 0 {
+		return nil
+	}
+
+	return strings.Split(vals, ",")
```

Comment:
Either here or on the backend, we should `Trim` each entry. This in case someone were to add a space after the comma: `ip:1.2.3.4, ip:2.3.4.5`

Reply:
I'd prefer to add trimming on the backend as that's a centralized point.